src/data.js

```diff
@@ -3793,6 +3793,17 @@ module.exports = [
     phone: 'iphone',
     tags: ['Web', 'Developer', 'JavaScript'],
   },
+  {
+    name: 'Ishaan Bedi',
+    description: 'Developer',
+    url: 'https://www.ishaanbedi.in/Uses',
```

Comment:
This URL redirects to `https://ishaanbedi.in/Uses-8f9cd4ad8dad4a35bcd9bcf30530279d` which fails to match the required pattern: `/(use|uses|using|setup|environment|^https:\/\/gist.github.com\/)/`